lib/index.js

```diff
@@ -77,6 +77,18 @@
  *   Change URLs (default: `defaultUrlTransform`)
  */
 
+/**
+ * @typedef HooksOptionsOnly
+ *   Configuration specifically for {@link MarkdownHooks}
+ * @property {ReactNode} [fallback]
+ *   A fallback node to render while the processor isn’t done processing the markdown.
```

Comment:
```suggestion
 *   Fallback content to render while the processor processing the markdown.
```

* More active
* Less negatives
* Content instead of node